.gitignore

```diff
@@ -102,4 +102,5 @@ certbot.log
 rerun.txt
 
 # Ignore VCR
-features/support/fixtures/cassettes/**
\ No newline at end of file
+features/support/fixtures/cassettes/**
```

Comment:
is it necessary to put this re-enabling of vcr in this pr?

Reply:
Not necessary for google cloud, but leaving them disabled can cause confusion in development (as it did with me).  I don't think the PR that took them out gave any reason for taking it out, so I'd rather keep it the way it has always been unless there's a good reason.

Reply:
actually looks like I just accidentally removed the last blank line, which doesn't seem to hurt anything